CHANGES.md

```diff
@@ -6,7 +6,30 @@ list](https://github.com/trinodb/trino-python-client/tags), the
 [README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
 and the [PyPI page](https://pypi.org/project/trino/).
 
-## Trino Python client 0.316.0
+## Release 0.317.0
+
+* Add support for creating tables containing `JSON` columns and reading and
+  writing to them with SQLAlchemy.
+  ([#194](https://trinodb/trino-python-client/issues/194))
+* Properly propagate query failures to the client when using `fetchone`.
+  ([#95](https://trinodb/trino-python-client/issues/95))
+* Fix queries returning a single row from sometimes appearing as failed on the
+  server. ([#220](https://trinodb/trino-python-client/issues/220))
+* Fix query failures when using SQLAlchemy `TableClause` by not performing
+  catalog lookup. ([#237](https://trinodb/trino-python-client/issues/237))
+* Fix errors when using prepared statements with Trino versions greater than or
+  equal to 398. ([#242](https://trinodb/trino-python-client/issues/242))
+
+### Breaking Changes
+
+* Make the `execute` method of the cursor block until at-least one row is
+ received. This means users no longer need to call `fetchone` or `fetchall` to
+ make sure query actually starts executing on the Trino server. Note that
+ results still need to be consumed by calling `fetchone` or `fetchall` to make
+ sure query isn't considered idle and terminated on the server.
+ ([#232](https://trinodb/trino-python-client/issues/232))
```

Comment:
```suggestion
* Block the `execute` method of the cursor until at least one row is
 received. Users no longer need to call `fetchone` or `fetchall` to
 ensure that a query starts executing on the Trino server. Note that
 results still need to be consumed by calling `fetchone` or `fetchall` to
 ensure that a query isn't considered idle and terminated on the server.
 ([#232](https://trinodb/trino-python-client/issues/232))
```

Reply:
No hyphen in at least, a few wording changes, and making sure we're properly using a/an/the.